app/controllers/agent/business/BusinessAccountingMethodController.scala

```diff
@@ -49,23 +47,23 @@ class BusinessAccountingMethodController @Inject()(val authService: AuthService,
 
   def show(isEditMode: Boolean): Action[AnyContent] = Authenticated.async { implicit request =>
     implicit user =>
-      require(optAccountingMethodAnswer and incomeSourceModelAnswer and matchTaxYearAnswer) { case optAccountingMethod ~ incomeSourceModel ~ matchTaxYear =>
+      require(optAccountingMethodAnswer and incomeSourceModelAnswer) { case optAccountingMethod ~ incomeSourceModel =>
         Future.successful(Ok(view(
           accountingMethodForm = AccountingMethodForm.accountingMethodForm.fill(optAccountingMethod),
           isEditMode = isEditMode,
-          backUrl = backUrl(isEditMode, incomeSourceModel, matchTaxYear)
+          backUrl = backUrl(isEditMode, incomeSourceModel)
         )))
       }
   }
 
   def submit(isEditMode: Boolean): Action[AnyContent] = Authenticated.async { implicit request =>
     implicit user =>
-      require(incomeSourceModelAnswer and matchTaxYearAnswer) { case incomeSourceModel ~ matchTaxYear =>
+      require(incomeSourceModelAnswer) { case incomeSourceModel =>
```

Comment:
I'm pretty sure when you require a single answe, you don't require the case
```suggestion
      require(incomeSourceModelAnswer) { incomeSourceModel =>
```